libs/sdk-ui-tests-e2e/cypress/tools/widget.ts

```diff
@@ -242,4 +242,34 @@ export class Widget {
         this.getElement().scrollIntoView();
         return this;
     }
+
+    getTooltipContents(index: number) {
```

Comment:
it seems duplicate with the one in chart.ts

Reply:
I've removed it